src/interfaces/Components.ts

```diff
@@ -283,6 +283,14 @@ export interface RadioButtonOptionsProps {
   value: string;
 }
 
+export interface PointsRulesProps {
+  enroll: number;
+  complete_chapter: number;
+  complete_course: number;
+  streak: number;
+  refer: number;
+}
```

Comment:
this is not an object but only a type. use different approach. we will send only one action type at once